src/inmanta/env.py

```diff
@@ -1094,14 +1094,41 @@ class VirtualEnv(ActiveEnv):
     Creates and uses a virtual environment for this process. This virtualenv inherits from the previously active one.
     """
 
+    _invalid_chars_in_path_re = re.compile(r"[\"$`]")
```

Comment:
This makes the set on invalid characters more clear.
```suggestion
    _invalid_chars_in_path_re = re.compile(r'["$`]')
```

Reply:
Oh, nice !